modules/rostests/apitests/crt/floor.c

```diff
@@ -0,0 +1,109 @@
+/*
+* PROJECT:     ReactOS api tests
+* LICENSE:     MIT (https://spdx.org/licenses/MIT)
+* PURPOSE:     Tests for floor / floorf
+* COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+*/
```

Comment:
```suggestion
 * PROJECT:     ReactOS api tests
 * LICENSE:     MIT (https://spdx.org/licenses/MIT)
 * PURPOSE:     Tests for floor / floorf
 * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
 */
```

Reply:
I already sent suggestion on this one above :smile: